blog/2024-06-14-using-table-component.md

```diff
@@ -0,0 +1,486 @@
+---
+author: Tom Gotsman
+date: 2024-06-12
+title: Using Table Component
+description: Describing main uses of the Table component with a Database
+image: 
+---
+
+```python exec
+from pcweb.pages.docs import library, database
+```
+
+The table component is one of the most useful components for visualizing, editing and working with data. In Reflex we have a built in `rx.table` component. This table efficiently displays data and allows embedding of various Reflex components, such as buttons, dropdowns, checkboxes, or forms, directly within table cells.
+
+In this blog we will link the data in our `rx.table` to an external database. The live version of the app we create in this blog can be found here: https://customer-data-app.reflex.run. The first thing that it is necessary to understand are `Tables`. 
```

Comment:
```suggestion
In this blog we will link the data in our `rx.table` to an external database. The live version of the app we create in this blog can be found here: https://customer-data-app.reflex.run. The first thing that it is necessary to understand are Tables. 
```

We shouldn't use code ticks around things that should not be literally typed as code.

Reply:
fixed